corehq/apps/es/es_query.py

```diff
@@ -394,6 +380,9 @@ def facets(self):
         results = namedtuple('facet_results', [f.name for f in facets])
         return results(**{f.name: f.parse_result(raw) for f in facets})
 
+    def __repr__(self):
+        return '{}({!r}, {!r})'.format(self.__class__.__name__, self.raw, self.query)
```

Comment:
Actually, are you sure `self.raw` isn't going to be massive?  Might be nice to stub that out.